src/pages/workspace/tags/TagSettingsPage.tsx

```diff
@@ -127,6 +131,14 @@ function TagSettingsPage({route, policyTags, navigation}: TagSettingsPageProps)
                             shouldShowRightIcon
                         />
                     </OfflineWithFeedback>
+                    <OfflineWithFeedback pendingAction={currentPolicyTag.pendingFields?.['GL Code']}>
+                        <MenuItemWithTopDescription
+                            title={currentPolicyTag['GL Code']}
+                            description={translate(`workspace.tags.glCode`)}
+                            onPress={navigateToEditGlCode}
```

Comment:
GL code should not be editable when there are accounting enabled for the account. Fixed here #49629.